cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1634,6 +1635,9 @@ private void handleCommand(ParsedCommandLine parsed) throws CommandFormatExcepti
                             invContext.getConfiguration().getOutputRedirection().close();
                         } catch (IOException ex) {
                             // Message must contain the Exception and the localized message.
+                            if (ex instanceof AccessDeniedException) {
+                                throw new CommandLineException(ex.getLocalizedMessage() + " (Permission denied)");
```

Comment:
When looking at the AcessDenied exception, I can see some interesting methods (getFile, getMessage, ...) Can we use them to build a user friendly message?

Reply:
I tried to get all the information from the exception, it does not look like we can improve: 
```
localized message - /dir/without/permission
message - /dir/without/permission
file - /dir/without/permission
other file - null
cause - null
toString - java.nio.file.AccessDeniedException: /dir/without/permission
reason - null
```

Reply:
@xstefank , thank-you to have look into this possibility, the safest message seems to me:
"Access denied to " + file
What do you think?

Reply:
@jfdenise the file may be null, I was thinking that `(message != null ? message : line) + (Permission denied)` would be sufficient in the rare case that exception did not included the file name

but definitely I can include your suggestion

Reply:
Yes, if we have a null file, then message should be the fallback. I would prefer "Access denied" instead of "Permission denied" this seems more in line with the reported problem.

Reply:
@jfdenise definitely, thanks for the review